install-dependencies.sh

```
@@ -224,6 +224,221 @@ function ensure_cargo() {
   done
   }
 
+# docker
+function ensure_docker() {
+  check_cmd docker && return
+
+  case $OS in
+  Linux)
+    local cmd=$(
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)